3rdParty/fuerte/src/H2Connection.cpp

```diff
@@ -846,14 +862,15 @@ bool H2Connection<T>::shouldStop() const {
 // ping ensures server does not close the connection
 template <SocketType T>
 void H2Connection<T>::startPing() {
+#if 0
```

Comment:
`startPing()` is useless now? intentional?

Reply:
I will re-enable it later again